src/parser/expression.js

```diff
@@ -385,6 +385,16 @@ pp.parseExprAtom = function (refShorthandDefaultPos) {
       }
       return this.finishNode(node, "Super");
 
+    case tt._import:
+      if (!this.hasPlugin("importFunctions")) this.unexpected();
+
+      node = this.startNode();
+      this.next();
+      if (!this.match(tt.parenL)) {
```

Comment:
Also codecoverage says we never hit this block, but as we do the same for super maybe be consistent and leave it here?